lib/commands/doctor/index.js

```diff
@@ -27,7 +27,10 @@ class DoctorCommand extends Command {
 
             if (
                 !argv.skipInstanceCheck &&
-                !(argv.categories && argv.categories.length === 1 && argv.categories[0] === 'install')
+                !(argv.categories &&
```

Comment:
instead of making this change, could we pass `skipInstanceCheck: true` to the doctor command in start?

Reply:
Done